packages/acceptance-tests/pkg-tests-specs/sources/protocols/git.test.js

```diff
@@ -0,0 +1,144 @@
+const {
+  fs: {readJson, readFile},
+  tests: {getPackageDirectoryPath},
+} = require('pkg-tests-core');
+const {parseSyml} = require('@berry/parsers');
+
+describe(`Protocols`, () => {
+  describe(`git:`, () => {
+    test(
+      `it should resolve a dependency with git tag via git protocol`,
+      makeTemporaryEnv(
+        {
+          dependencies: {[`util-deprecate`]: `git://github.com/TooTallNate/util-deprecate.git#v1.0.1`},
+        },
+        async ({path, run, source}) => {
+          await run(`install`);
+
+          await expect(readJson(`${path}/package.json`)).resolves.toMatchObject({
+            dependencies: {
+              [`util-deprecate`]: `git://github.com/TooTallNate/util-deprecate.git#v1.0.1`,
+            },
+          });
+
+          const content = await readFile(`${path}/yarn.lock`, `utf8`);
+          const lock = parseSyml(content);
+
+          await expect(lock).toMatchObject({
+            [`util-deprecate@git://github.com/TooTallNate/util-deprecate.git#v1.0.1`]: {
+                "version": "1.0.1"
+            }
+          });
```

Comment:
Can you add a check that the package can be required, like in [here](https://github.com/yarnpkg/berry/blob/master/packages/acceptance-tests/pkg-tests-specs/sources/basic.test.js#L15)?

Reply:
When I added this to the test

```
await expect(source(`require('util-deprecate')`)).resolves.toMatchObject({
            name: `util-deprecate`,
            version: `1.0.1`,
          });
```

I received:

```
expect(received).resolves.toMatchObject()

    Received promise rejected instead of resolved
    Rejected to value: [SyntaxError: Unexpected token u in JSON at position 0]

      31 |           });
      32 | 
    > 33 |           await expect(source(`require('util-deprecate')`)).resolves.toMatchObject({
         |                 ^
      34 |             name: `util-deprecate`,
      35 |             version: `1.0.1`,
      36 |           });

      at expect (../../.yarn/cache/expect-npm-24.5.0-c67446aa4c28bbfac0c9e221e9cd6ebbd21da6df4c9c686ff86ea72852a7c5b9.zip/node_modules/expect/build/index.js:132:15)
      at expect (pkg-tests-specs/sources/protocols/git.test.js:33:17)
```

😬 Do you know why is that happening? 🤔

Reply:
`require('util-deprecate/package.json`)` 😉